pnpm-lock.yaml

```diff
@@ -5774,6 +5780,8 @@ snapshots:
 
   '@types/estree@1.0.0': {}
 
+  '@types/estree@1.0.6': {}
```

Comment:
I see some of these can be dedupe, but I only know one way which is to run `pnpm dedupe`, but it gives lots of other unrelated changes too, so probably best do it centrally and not in this PR